File: routes/book_routes.py
from flask import Blueprint
from controllers.book_controller import (
    create_book,
    list_books,
    update_book,
    delete_book
)

book_routes = Blueprint("book_routes", __name__)

book_routes.route("/", methods=["POST"])(create_book)
book_routes.route("/", methods=["GET"])(list_books)
book_routes.route("/<int:id>", methods=["PUT"])(update_book)
book_routes.route("/<int:id>", methods=["DELETE"])(delete_book)

File: routes/book_routes.js
// /routes/bookRoutes.js

const express = require('express');
const router = express.Router();
const bookController = require('../controllers/bookController');


router.post('/books', bookController.createBook); // Criar livro
router.get('/books', bookController.getAllBooks); // Listar livros
router.put('/books/:id', bookController.updateBook); // Atualizar livro
router.delete('/books/:id', bookController.deleteBook); // Excluir livro

module.exports = router;
